src/TableColumn/Image.php

```diff
@@ -4,29 +4,13 @@
 
 namespace atk4\ui\TableColumn;
 
+if (!class_exists(\SebastianBergmann\CodeCoverage\CodeCoverage::class, false)) {
+    'trigger_error'('Use atk4\ui\Table\Column\Image instead', E_USER_DEPRECATED);
+}
+
 /**
- * Column for formatting image.
+ * @deprecated will be removed jun-2021
```

Comment:
I think we should target for 1/2 year instead - this is to allow enought releases in between for smooth transition, otherwise we want to remove it asap